.kokoro/e2e_tests-optional_container_engine-datastore.cfg

```diff
@@ -0,0 +1,22 @@
+# Format: //devtools/kokoro/config/proto/build.proto
```

Comment:
filename: container --> kubernetes

Reply:
These require further changes to Kokoro configs. I want to merge this as-is (while the builds are passing) and make those changes in a separate PR (to prevent this PR growing even larger).